go.mod

```diff
@@ -1,17 +1,17 @@
 module github.com/sourcegraph/lsif-go
 
-go 1.12
+go 1.15
 
 require (
 	github.com/alecthomas/kingpin v2.2.6+incompatible
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
 	github.com/alecthomas/units v0.0.0-20210208195552-ff826a37aa15 // indirect
 	github.com/efritz/pentimento v0.0.0-20190429011147-ade47d831101
-	github.com/google/go-cmp v0.5.2
+	github.com/google/go-cmp v0.5.4
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/slimsag/godocmd v0.0.0-20161025000126-a1005ad29fe3
-	github.com/sourcegraph/lsif-protocol v1.0.0
+	github.com/sourcegraph/sourcegraph/enterprise/lib v0.0.0-20210301210707-8f7f90511f7c
```

Comment:
Same point as in the other PR : )

Reply:
I may have completely missed what repo those last 3 comments were in :facepalm: my bad! ignore them